src/services/AsyncStorage.ts

```diff
@@ -10,6 +10,7 @@ const FAVORITES_KEY = 'favorites'
 const PROGRESS_KEY = 'progress'
 const SENTRY_KEY = 'sentryTracking'
 const CMS_KEY = 'cms'
+const MODE_KEY = 'devmode'
```

Comment:
I think naming could be better here. Maybe just `DEV_MODE_KEY` oder `CHEAT_MODE_KEY`?